scripts/mypy-failing.txt

```diff
@@ -7,6 +7,7 @@ pytensor/compile/sharedvalue.py
 pytensor/graph/rewriting/basic.py
 pytensor/ifelse.py
 pytensor/link/basic.py
+pytensor/link/c/cmodule.py
```

Comment:
I think you'll have to ignore this pre-commit failures locally until we figure out. On the CI it doesn't behave like in your local machine and these changes break it. You can add `no-verify` when commiting from the terminal (or unselect `run pre-commit hooks` when commiting from pycharm).